### DISCUSSION TITLE
How can I delete this discussion?

### DISCUSSION THREAD

**Discussion by arturober** [community/community#39270] [Other Feature Feedback, Questions, & Ideas]

**Comment by DSDmark** [comment]
Hi @arturober,

Click on three dots which is on your right hand side in this discussion there is delete option.

![Screenshot_2022-11-16-23-42-42-303_com brave browser](https://user-images.githubusercontent.com/85744816/202260109-f5372f85-865c-4fc2-84a3-5ee074430f45.jpg)

**Reply by arturober** [reply]
That only seems to work with the answers...
![image](https://user-images.githubusercontent.com/7542665/202262643-40b511d0-f149-4768-9a94-2b9a796c2016.png)

**Comment by official-mywebprojects** [comment]
For now, only repository admins and maintainers can delete a discussion or an entire conversation on GitHub.
You can read more on discussion deletion here

**Comment by official-mywebprojects** [comment]
https://docs.github.com/en/discussions/managing-discussions-for-your-community/managing-discussions

**Comment by official-mywebprojects** [comment]
https://docs.github.com/en/discussions/managing-discussions-for-your-community/managing-discussions
